applicationconfig/client/README.md

```diff
@@ -0,0 +1,95 @@
+# Azure App Configuration client library for Java
+Azure App Configuration is a managed service that helps developers centralize their application configurations simply and securely.
+
+Use the client library for App Configuration to:
+- Add a new setting configuration
+- Update an existing setting configuration
+- Retrieve a setting configuration
+- Delete a setting configuration
+
+[Source code]() | [Package (Maven)]() | [API reference documentation]() | [Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
+
+## Getting started
+
+### Prerequisites
+
+- JDK 8 or later
+- Apache Maven 3 or above
+- Azure Service Principal
+- App Configuration Store
+
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-app-configuration</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+### Authenticate the client
+
+```java
+String connectionString = "endpoint={endpoint_value};id={id_value};secret={secret_value}";
+ConfigurationClient client = ConfigurationClient.builder()
+        .credentials(new ConfigurationClientCredentials(connectionString)
+        .build();
+```
+
+Using the Azure web portal to open your App Configuration Store's Access Key section, choose one of the two connection strings and use it to replace the string value above.
+
+## Key concepts
+
+### [Key-value store](https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-key-value)
+
+Configurations are stored using key-value pairs. Keys must be unique for each configuration so it's common practice to organize keys into hierarchical namespaces using delimiting characters, for example AppName:Service1:DB:Endpoint. Optionally a label attribute may be given to a key to differentiate two configurations, for example having different versions of the configuration. Values are stored as unicode strings.
```

Comment:
I'm not a fan of this description. It misleads on how to set keys. For example, you are [omitting ](App Configuration treats keys as a whole. It doesn't parse keys to figure out how their names are structured or enforce any rule on them.) that the service doesn't do any parsing and therefore, the customer will have to do it.

Also, you are referring to what a key is, but the title talks about a key-value store, which in our SDK language means a Configuration Setting, which includes way more information than a key... 
Something like [this](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane#configuration-setting) might be better